docs/server_functional_requirements.md

```diff
@@ -81,19 +81,14 @@ Minimum required fields, followed by a JSON example:
   * Description: 2 letter country to identify the region(s) a key is valid for.
 * `appPackageName` (**REQUIRED**)
   * Type: string
-  * Constraints:
-    * For Android device, this must match the device `appPackageName` in the
-      SafetyNet verification payload
   * Description: name of the application bundle that sent the request.
 * `platform` (**REQUIRED**)
```

Comment:
Platform is also deprecated now - it was only needed to differentiate between DC and SN calls.

Reply:
Done